code/client/munkilib/munkicommon.py

```diff
@@ -842,6 +843,15 @@ def getFirstPlist(textString):
     return (textString[plist_start_index:plist_end_index],
             textString[plist_end_index:])
 
+def getAppleLanguages():
+    user = getconsoleuser()
+    userHome = pwd.getpwnam(user).pw_dir
+    try:
+        globalPreferences = FoundationPlist.readPlist(userHome + "/Library/Preferences/.GlobalPreferences.plist")
```

Comment:
This is not-great Python. Use the os.path routines:

os.path.join(userHome, 'Library/Preferences/.GlobalPreferences.plist')

Additionally, it's generally _not_ a great idea to bypass cfprefs and directly read user preference plist files...